src/main/java/com/clanevents/ClanEventsPanel.java

```diff
@@ -201,6 +187,34 @@ public void mousePressed(MouseEvent e) {
 
         //Add the plugin's main panel
         this.add(ssArea, BorderLayout.NORTH);
+        service.addListener(this);
+        if (service.state.equals(State.IDLE)) service.refreshData();
+    }
+
+    @Override
+    public void propertyChange(PropertyChangeEvent evt) {
+        if (evt.getPropertyName().equals("state") && evt.getNewValue() != evt.getOldValue()) {
+            System.out.printf("State changing: old: %s, new: %s%n", evt.getOldValue(), evt.getNewValue());
```

Comment:
```suggestion
            //System.out.printf("State changing: old: %s, new: %s%n", evt.getOldValue(), evt.getNewValue());
```
It's a bit annoying when testing your own plugins to have your console spammed with println or printf from installed plugins. Would be lovely if you could comment it out!

Reply:
removed prints.